webapi/src/test/scala/org/knora/webapi/store/iiif/impl/IIIFServiceMockSipiImpl.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.store.iiif.impl
+
+import akka.actor.{Actor, ActorLogging, ActorSystem}
+import akka.http.scaladsl.util.FastFuture
+import org.knora.webapi.exceptions.SipiException
+import org.knora.webapi.messages.store.sipimessages._
+import org.knora.webapi.messages.v2.responder.SuccessResponseV2
+import org.knora.webapi.settings.KnoraDispatchers
+import org.knora.webapi.util.ActorUtil._
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success, Try}
+import org.knora.webapi.store.iiif.api.IIIFService
+import zio._
+
+/**
+ * Can be used in place of [[IIIFServiceSipiImpl]] for tests without an actual Sipi server, by returning hard-coded
+ * responses simulating responses from Sipi.
+ */
+case class IIIFServiceMockSipiImpl(str: String) extends IIIFService {
+
+  /**
+   * A request to [[MockSipiConnector]] with this filename will always cause the responder to simulate a Sipi
```

Comment:
is `MockSipiConnector` still the correct name?

Reply:
changed